File: nombre.java
import java.util.*;

public class nombre {

    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        String nombre,apellido;
        System.out.print("Cual es tu nombre? ");
        nombre = sc.nextLine();
        System.out.print("Cual es tu apellido? ");
        apellido = sc.nextLine();
        sc.close();
        System.out.println("Tu nombre completo es: " + nombre + " " + apellido);
    } 

}

File: nombre.go
package main

import (
	"fmt"
)

func main() {

	var name, apellido string

	fmt.Print("Escribe tu nombre: ")
	fmt.Scanf("%s", &name)

	fmt.Print("Escribe tu apellido: ")
	fmt.Scanf("%s", &apellido)

	fmt.Print("Tu nombre completo es: " + name + " " + apellido + " \n")

}
